tutorials/buoyantSimpleFoam-aster/config.comm

```diff
@@ -0,0 +1,34 @@
+settings = \
+{'base-path': '/home/michel/Git/code_aster-adapter/tutorials/buoyantSimpleFoam-aster',
```

Comment:
```suggestion
{'base-path': 'path/to/code_aster-adapter/tutorials/buoyantSimpleFoam-aster',
```

Reply:
Set this environment in the Allrun script.

Reply:
For example: `export CA_BASE_PATH=$(pwd)`

Reply:
For now pushed a code with the 'path/to/' solution

Reply:
Maybe you can try accessing the environment variable using the `os` Python package: `os.environ['CA_BASE_PATH']`. See [StackOverflow](https://stackoverflow.com/questions/4906977/how-to-access-environment-variable-values).

Reply:
I accessed the environment in adapter.comm with `os.environ['CA_BASE_PATH']`